### ISSUE TITLE
Add margins to generated GIFs

### ISSUE THREAD

**Issue by SkypLabs** [asciinema/agg#29]
Hi,

This issue follows up on https://github.com/asciinema/asciicast2gif/issues/82.

It would be nice to be able to add margins to the generated GIFs. My use case is to improve the rendering of the [GitHub repositories' social media previews](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview).

The template that GitHub provides suggests to add 40px margins around the image.

Cheers.

**Comment by sickill** [comment]
Makes sense to have option like this. If we were to add this I believe it should be opt-in, and customizable, e.g. `--margin-width 40` or similar.

**Comment by SkypLabs** [comment]
> Makes sense to have option like this. If we were to add this I believe it should be opt-in, and customizable, e.g. --margin-width 40 or similar.

Yes, it's exactly what I had in mind 👍
